debezium-core/src/main/java/io/debezium/jdbc/JdbcValueConverters.java

```diff
@@ -424,22 +389,62 @@ protected Object convertTimeWithZone(Column column, Field fieldDefn, Object data
         });
     }
 
-    protected Object convertTime(Column column, Field fieldDefn, Object data) {
-        if (adaptiveTimeMicrosecondsPrecisionMode) {
-            return convertTimeToMicrosPastMidnight(column, fieldDefn, data);
+    protected Object convertDate(Column column, Field fieldDefn, Object data) {
+        switch (temporalPrecisionMode) {
+            case ISOSTRING:
+                return convertDateToUtcIsoString(column, fieldDefn, data);
+            case ADAPTIVE_TIME_MICROSECONDS:
+            case NANOSECONDS:
+            case MICROSECONDS:
```

Comment:
this were missing and catched by test. added them.

`NANOSECONDS` and `MICROSECONDS` will generate `EpochDays` for `DATE` values.